mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -5378,12 +5418,11 @@ pub struct HistoricalOrder {
     conf_settings: Option<OrderConfirmationsSettings>,
 }
 
-pub async fn orders_kick_start(ctx: &MmArc) -> Result<HashSet<String>, String> {
+pub async fn orders_kick_start(ctx: &MmArc, db_id: Option<&str>) -> Result<HashSet<String>, String> {
```

Comment:
Similar to swaps kickstarts, we need to only kickstart orders if the coin is activated with the same `db_id` as the order's `db_id` ref.
https://github.com/KomodoPlatform/komodo-defi-framework/blob/9a8494e02b5ebf2a40b66ffa9028d3d6a730ac71/mm2src/mm2_main/src/lp_swap.rs#L1545-L1601

Reply:
Sorry if I wasn't clear about this. I meant that if the other coin of the order was with a different `db_id`. Similar to what we done with swaps before if you remember.